src/main/java/uk/ac/cam/cl/dtg/segue/api/Constants.java

```diff
@@ -375,22 +375,32 @@ public String toString() {
     public static final Integer CONNECTION_POOL_MIN_MIN_IDLE = 10;
 
     public enum TimeInterval {
-        TWO_YEARS(2, 0, 0, 0, 0, 0),
-        SIX_MONTHS(0, 6, 0, 0, 0, 0),
-        NINETY_DAYS(0, 0, 90, 0, 0, 0),
-        THIRTY_DAYS(0, 0, 30, 0, 0, 0),
-        SEVEN_DAYS(0, 0, 7, 0, 0, 0);
+        twoYears(2, 0, 0, 0, 0, 0),
+        sixMonths(0, 6, 0, 0, 0, 0),
+        ninetyDays(0, 0, 90, 0, 0, 0),
+        thirtyDays(0, 0, 30, 0, 0, 0),
+        sevenDays(0, 0, 7, 0, 0, 0);
 
         private final PGInterval interval;
 
         TimeInterval(final int years, final int months, final int days, final int hours, final int minutes, final double seconds) {
             this.interval = new PGInterval(years, months, days, hours, minutes, seconds);
         }
-        public PGInterval getPGInterval() {
+
+        public PGInterval getInterval() {
             return this.interval;
         }
+
+        public static PGInterval getPGInterval(final String timeRange) {
```

Comment:
Technically the method probably is not needed but adding redundancy code seems to be a good thing. It could also be possible that the code now can be reused with different timeRanges as well.

Reply:
This is the same case as https://github.com/isaaccomputerscience/isaac-react-app/pull/47#discussion_r1315657557

Reply:
Changes have been made to address this